tests/Domain/DomainObjectTest.php

```diff
@@ -247,6 +249,14 @@ public function parserDataSet(): array
                 include __DIR__ . '/data/notification_poll_invalid_count.php',
                 TypeError::class,
             ],
+            'valid provider' => [
+                Provider::class,
+                include __DIR__ . '/data/provider_valid.php',
+            ],
```

Comment:
Can you also add cases with only required fields, and then cases with invalid data?

Reply:
I added a few more cases for Provider and Downtime objects: https://github.com/sandwave-io/realtimeregister-php/pull/22/commits/9743603880caf5d6044ac7abc219848654014498